Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine
+
+ARG VIPS_VERSION
+
+RUN apk update
+
+RUN apk add build-base pkgconfig glib-dev gobject-introspection-dev expat-dev tiff-dev libjpeg-turbo-dev libexif-dev giflib-dev librsvg-dev lcms2-dev libpng-dev orc-dev libwebp-dev libheif-dev
```

Comment:
libheif-dev brings in x265-dev which contains patent-encumbered software.

Reply:
Some of these dependencies are build-time only so can be removed from the final image. The `apk` utility allows the creation of virtual groups of dependencies that can be used for this purpose. This would allow the separation of e.g. `tiff` (required at build and runtime) from `tiff-dev` (required only at build time).

Reply:
Did the changes